### PR TITLE
feat/migrate-antd5-remove-style-import-codemo

### PR DESCRIPTION
**Pull request by DmytroHryshyn**

### CODE REVIEW

**Comment by DmytroHryshyn** [comment]
Current dependencies on/for this PR:
* main
  * **PR #516** <a href="https://app.graphite.dev/github/pr/intuita-inc/codemod-registry/516?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/intuita-inc/codemod-registry/516?utm_source=stack-comment).

File: cjs-builder/index.ts [hunk 1]
```diff
@@ -28,7 +48,8 @@
 	const options: Parameters<typeof esbuild.build>[0] = {
 		entryPoints: [relativeInputFilePath],
 		bundle: true,
-		packages: 'external',
+		// packages: 'external',
+		external: getExternal(packageJSON),
```
Comment (grzpab) [inline cjs-builder/index.ts:52]: actually we might pass the external package names as the yargs options to the CJS builder OR
We actually remove both `packages` and `external` and allow `esbuild` to bundle the normal dependencies. We will have to move all `dependencies` to `devDependencies`

Comment (DmytroHryshyn) [reply cjs-builder/index.ts:52]: let do this one
>We actually remove both packages and external and allow esbuild to bundle the normal dependencies. We will have to move all dependencies to devDependencies

File: cjs-builder/package.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
 		"cjs-builder": "./dist/index.js"
 	},
 	"scripts": {
+		"build": "esbuild ./index.ts --bundle --platform=node --target=node16 --format=esm --outfile=./dist/index.js --external:esbuild",
```
Comment (grzpab) [inline cjs-builder/package.json:9]: why not `tsc`?

File: cjs-builder/index.ts [hunk 3]
```diff
@@ -3,6 +3,9 @@ import esbuild from 'esbuild';
 import { mkdir, readFile, writeFile } from 'node:fs/promises';
 import { dirname, join } from 'node:path';
 
+// list of packages that should be bundled to the codemod (e.g codemod internal utils)
+const EXTERNAL_DEPENDENCIES = ['jscodeshift', 'ts-morph'];
```
Comment (grzpab) [inline cjs-builder/index.ts:7]: hmm what about `@intuita-inc/filemod`?

Comment (DmytroHryshyn) [reply cjs-builder/index.ts:7]: we only import types from it, so its fine

**Review by grzpab — APPROVED** [review]

**Pull request merged**